bitsv/wallet.py

```diff
@@ -153,14 +170,25 @@ def __init__(self, wif=None):
         self.balance = 0
         self.unspents = []
         self.transactions = []
-        self.network = 'main'
-        self.network_api = network_api_main  # instantiated above - avoids multiple unnecessary instances for many Keys
+        self.network = network
+
+        # Standard network_api_main/test/stn objects are instantiated at top of this file as globals.
+        # Avoids multiple unnecessary instances of these in the case of many Keys
+        if network == 'main':
+            self.network_api = network_api_main
+            self.version = 'main'
+        elif network == 'test':
+            self.network_api = network_api_test
+            self.version = 'test'
+        elif network == 'stn':
+            self.network_api = network_api_stn
+            self.version = 'test'
```

Comment:
Similar comment here. This should have a comment at a minimum as it's not very intuitive.

Reply:
Thanks. I'll see what I can do on Saturday. Im already having ideas of how to greatly simplify...